playwright-e2e/playwright.config.ts

```diff
@@ -38,7 +37,7 @@ const testConfig: PlaywrightTestConfig = {
     }
   },
   /* Run tests in files in parallel */
-  fullyParallel: true,
+  // fullyParallel: true,
```

Comment:
I agree with disabling parallel testing. Instead of leaving a commented out line there, can it be either removed entirely, or explicitly set to `false`?

Reply:
Good call. I'll set it to `false`.